modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Updated the image dialog to display the class list dropdown as full-width if the caption checkbox is not present #TINY-6400
```

Comment:
This isn't an `Added`/New Feature, it's a `Changed` or `Improved` at best, so please make sure to move this to the appropriate heading.

- [ ] Move changelog entry to correct heading

Reply:
```suggestion
- Updated the `image` dialog to display the class list dropdown as full-width if the caption checkbox is not present #TINY-6400
```
Also just another minor thing, as image is the plugin name it should be in back ticks.

Reply:
updated